openassessment/templates/openassessmentblock/staff_debug/student_info.html

```diff
@@ -236,6 +236,6 @@ <h3 class="title">{% trans "Example-Based Assessment" %}</h3>
         {% endif %}
     </div>
     {% else %}
-        {% trans "Couldn't find a response for this student." %}
+        {% trans "Couldn't find a response for this learner." %}
```

Comment:
Nit: I believe that our style guide discourages contractions. @catong, is that right?